commons/ava_services/gecko_service_tls_identities.go

```diff
@@ -0,0 +1,86 @@
+package ava_services
+
+const STAKER_1_NODE_ID = "7Xhw2mDxuDS44j42TCB6U5579esbSt3Lg"
```

Comment:
Need a biiiiig fat TODO explaining what this is, and why we have hardcoded certs (especially private keys!!!) in here

Reply:
done